code/modules/uplink/uplink_items.dm

```diff
@@ -264,6 +264,14 @@ GLOBAL_LIST_INIT(illegal_tech_blacklist, typecacheof(list(
 	cost = 30
 	purchasable_from = UPLINK_NUKE_OPS
 
+/datum/uplink_item/bundles_TC/machoman
+	name = "The Macho Wrestler Bundle"
+	desc = "For the STRONGEST operatives! Master your wrastling abilitys with this bundle. \
+			Contains: A wrestling belt, for only the strongest, a adrenalin implanter, a mask to strike fear into your enemys, and a combat hypospray!"
```

Comment:
Contains: A wrestling belt, only for the strongest machos, an adrenaline implanter, a luchador mask to strike fear into your enemies, and a combat hypospray!"